packages/lib/src/artifacts/Contract.ts

```diff
@@ -119,7 +94,7 @@ function _wrapContractInstance(schema: any, instance: Web3Contract): Contract {
   instance.at = function(address: string): Contract | never {
     if (!ZWeb3.isAddress(address)) throw new Error('Given address is not valid: ' + address);
     const newWeb3Instance = instance.clone();
-    newWeb3Instance._address = address;
+    newWeb3Instance['_address'] = address;
```

Comment:
Why does this change have to be made? Also, why is setting `options` not enough?

Reply:
`_address` is not public in the Contract type, and its handled as a separate property